docs/tasks/administer-cluster/assign-pods-nodes.md

```diff
@@ -0,0 +1,88 @@
+---
+---
+
+{% capture overview %}
+This page shows how to assign a Kubernetes Pod to a particular node in a
+Kubernetes cluster.
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* Install [kubectl](http://kubernetes.io/docs/user-guide/prereqs).
+
+* Create a Kubernetes cluster, including a running Kubernetes
+  API server. One way to create a new cluster is to use
+  [Minikube](/docs/getting-started-guides/minikube).
```

Comment:
I thought minikube only supported a single node at the moment.  Does this really make sense as a suggestion for a guide on "assigning pods to specific nodes"?